init.sh

```diff
@@ -1,6 +1,17 @@
 #!/bin/bash
 
-NAME=coffeaenv
+source local.sh
+NAME=$1
```

Comment:
This line breaks our condor bash script here:
https://github.com/cms-svj/t-channel_Analysis/blob/kjp/setup/condor/run_Analyzer_condor.sh#L27
Not sure how to fix it nicely other than to hard code the NAME in this script.

Reply:
The $1 from the run_Analyzer_condor.sh script gets passed into the init.sh script when running on condor, so it doesn't source the environment properly.

Reply:
I think changing it to `source init.sh ""` will work.